From 24832bf3255db431aa6c8af6ebff2b0fa7e4ff7e Mon Sep 17 00:00:00 2001
From: LuK1337 <priv.luk@gmail.com>
Date: Thu, 30 May 2024 10:52:43 +0200
Subject: [PATCH 2/2] Disable publication signing

---
 wrappers/android/zxingcpp/build.gradle.kts | 7 -------
 1 file changed, 7 deletions(-)

diff --git a/wrappers/android/zxingcpp/build.gradle.kts b/wrappers/android/zxingcpp/build.gradle.kts
index 81a9d3de..80e26fc0 100644
--- a/wrappers/android/zxingcpp/build.gradle.kts
+++ b/wrappers/android/zxingcpp/build.gradle.kts
@@ -118,10 +118,3 @@ publishing {
         }
     }
 }
-
-signing {
-    val signingKey: String? by project
-    val signingPassword: String? by project
-    useInMemoryPgpKeys(signingKey, signingPassword)
-    sign(publishing.publications)
-}
-- 
2.45.1

